spatialprofilingtoolbox/cggnn/scripts/upload_importance.py

```diff
@@ -0,0 +1,49 @@
+"""Uploads importance score output from a cg-gnn instance to the local db."""
+
```

Comment:
Elsewhere, this pattern was for grouping imports except that there is no need for a blank line between the module docstring and the first standard library import. In other places where there is a blank line after the module docstring, it is supposed indicate that there are no standard library imports.

Reply:
My personal preferred style is a line break after all docstrings, I think it looks nicer. I stopped doing so in some places after pydocstyle complained about line breaks after _method_ docstrings, but it doesn't complain about adding a line break after module docstrings.

That said, feel free to change if this isn't SPT style.

Reply:
Ah I see, so it is not import-related. The pattern I described above was just something I made up when I was trying to choose a convention. I don't have much of a justification for it.
I'm fine with any choice here, blank line or not after any docstrings. I am happy to conform to your preference here from now on.